AudioKitSynthOne/DSP/Audio Unit/S1AudioUnit.h

```diff
@@ -101,6 +97,16 @@ typedef struct S1ArpBeatCounter {
 - (void)resetDSP;
 - (void)resetSequencer;
 
+// S1TuningTable protocol
+- (void)setTuningTable:(float)frequency index:(int)index;
+- (float)getTuningTable:(int)index;
```

Comment:
I'd rephrase this to `getTuningTableFrequency:(int)noteIndex;` .. others might be confused else.

Reply:
Will update